app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx

```diff
@@ -168,19 +173,25 @@ export class LogToolbar extends Component {
     } = this.props;
     return (
       <div className={cx('log-toolbar')}>
-        <Breadcrumbs
-          descriptors={breadcrumbs}
-          togglerEventInfo={LOG_PAGE_EVENTS.plusMinusBreadcrumb}
-          breadcrumbEventInfo={LOG_PAGE_EVENTS.ITEM_NAME_BREADCRUMB_CLICK}
-          allEventClick={LOG_PAGE_EVENTS.ALL_LABEL_BREADCRUMB}
-          onRestorePath={restorePath}
-        />
+        <div className={cx('breadcrumbs-wrapper')}>
+          <Breadcrumbs
+            descriptors={breadcrumbs}
+            togglerEventInfo={LOG_PAGE_EVENTS.plusMinusBreadcrumb}
+            breadcrumbEventInfo={LOG_PAGE_EVENTS.ITEM_NAME_BREADCRUMB_CLICK}
+            allEventClick={LOG_PAGE_EVENTS.ALL_LABEL_BREADCRUMB}
+            onRestorePath={restorePath}
+          />
+        </div>
         <div className={cx('action-buttons')}>
           {logViewMode === DETAILED_LOG_VIEW ? (
             <>
               {!debugMode && (
                 <InputCheckbox onChange={this.changeHistoryLineMode} value={includeAllLaunches}>
-                  {intl.formatMessage(messages.historyAllLaunchesLabel)}
+                  {intl.formatMessage(
+                    window.innerWidth > SCREEN_XL_MAX
+                      ? messages.historyAllLaunchesLabel
+                      : messages.historyShortAllLaunchesLabel,
+                  )}
```

Comment:
As we need to use short label only on mobile, please use corresponding `SCREEN_XS_MAX` breadkpoint.

Reply:
Also you may provide a custom class to `InputCheckbox` component to control wrappig behaviour. F.e. adding the CSS property `white-space: nowrap;` to this field will always keep all words in one line.